common/dynamicconfig/constants.go

```diff
@@ -250,6 +250,9 @@ const (
 	// FrontendEnableBatcher enables batcher-related RPCs in the frontend
 	FrontendEnableBatcher = "frontend.enableBatcher"
 
+	// FrontendEnableUpdateWorkflowExecution enables UpdateWorkflowExecution API in the frontend.
+	FrontendEnableUpdateWorkflowExecution = "frontend.enableUpdateWorkflowExecution"
```

Comment:
Add comment that this feature is in development and is not ready to use in prod.